docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -122,6 +122,11 @@ Examples:
 |
 |List of relative URLs, a seed URL is a URL that is fetched by the crawler to extract new URLs in it and follow them for crawling.
 
+|`transformer.from-headless-crawling.allowed-extensions`
+|Comma-separated list of values
+|`htm,html,php,asp,aspx,jsp,md`
+|Allowed list of file extensions in URL. The crawler will ignore all URLs referring to files with other extensions. URLs without extension will always be included in crawling.
```

Comment:
it makes sense to put a note that extensions in this property are case-insensitive